conf_files/pocs.yaml

```diff
@@ -64,3 +65,8 @@ messaging:
 #         output_timestamp: False
 
 state_machine: simple_state_table
+# Default coordinates to take evening flats
+flat_field:
+    twilight:
+        alt: 70
+        az: 120
```

Comment:
How are these selected? Seems like we should compute the az from the sun.

Reply:
Yes, probably should. That change might need to come when we add the morning flats so are having to compute it anyway.